### PR TITLE
Redirect windows image requests

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/135251

### CODE REVIEW

File: etc/nginx/vhosts.d/openqa-locations.inc [hunk 1]
```diff
@@ -56,4 +56,5 @@ location / {
     proxy_set_header X-Forwarded-Server $host;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
+    rewrite ^/tests/.*/asset/.*/[wW]in.*$ https://www.microsoft.com/software-download/windows10ISO last;
```
Comment (baierjan) [inline etc/nginx/vhosts.d/openqa-locations.inc:59]: Wasn't that asset**s** in the original rule?

Comment (baierjan) [inline etc/nginx/vhosts.d/openqa-locations.inc:59]:
```suggestion
    rewrite ^(/tests/.*)?/assets?/.*/[wW]in.*$ https://www.microsoft.com/software-download/ last;
```

Comment (perlpunk) [reply etc/nginx/vhosts.d/openqa-locations.inc:59]: See my comment in the ticket. The `/assets` url was commented out, so I didn't port that one.

Comment (baierjan) [reply etc/nginx/vhosts.d/openqa-locations.inc:59]: Makes sense, I was just looking at the original ticket description.

**Pull request closed without merging**